src/contexts/Localisation.tsx

```diff
@@ -88,7 +88,7 @@ export function LanguageProvider({
   const [languageState, setLanguageState] = useState<LanguageState>({
     languageOptions,
     selectedLanguage: initSelectedLanguage,
-    strings: {} as LanguageStrings,
+    strings,
```

Comment:
Not related, but this just makes sure we start with the default strings available, otherwise we're seeing "LABEL_LOADING" in the UI when first loading, since the language files haven't been fetched yet. So now we'll at least get a sensible default.